packages/semcom-components/lib/components/input.component.ts

```diff
@@ -9,21 +9,21 @@ export class InputComponent extends BaseComponent {
    * The input field.
    */
   @query('#content')
-  content: HTMLInputElement;
+  content!: HTMLInputElement;
```

Comment:
what does this do exactly? ( just curious )

Reply:
it is to make sure the variables are never null or undefined
https://lit.dev/docs/components/shadow-dom/

Reply:
But it actually prevents TS from doing the type check. In the case of LitElement, it might be semi-safe, since you "can be sure" that the template contains the element (if you're confident that you never make mistakes, which I am not of myself, for example)... but it will lead to runtime errors when trying to access the element before rendering, or when using it wrongly with state (see other comment).

Reply:
We need to take a look at this again and decide what (and why) we want to do this.

Reply:
Ah, @arnevandoorslaer, I forgot about this... can you take another look at it tomorrow? I'd actually prefer to never use that syntax, and simply check for undefined values when they're used (explicitly or using `?.` or `??` operators), but maybe you can convince me if you really think they have a huge advantage.